src/components/app/flyouts/settings/settings.js

```diff
@@ -116,6 +154,29 @@ export class Settings extends LinkedComponent {
             <Flyout.CloseBtn onClick={onClose} />
           </Flyout.Header>
           <Flyout.Content className="settings-workflow-container">
+            <Section.Container collapsable={false}>
+              <Section.Header>{t('settingsFlyout.sendDiagnosticsHeader')}</Section.Header>
+              <Section.Content className="diagnostics-contnent">
+                <label>
+                  {this.props.t('settingsFlyout.sendDiagnosticsText')}
+                </label>
+                <br />
```

Comment:
Avoid using `<br />`, use css to add spacing

Reply:
This is simplistic. Otherwise, I have to do deal with flex and alignment.

Reply:
We still don't want to add `<br/>`.  We have to deal with flex and alignment everywhere else in the application CSS. This is no different.